src/main/initVerticalMod.F90

```diff
@@ -132,10 +198,6 @@ subroutine initVertical(bounds, glc_behavior, thick_wall, thick_roof)
     ! The distinction between "shallow" and "deep" bedrock is not made explicitly
     ! elsewhere. But, since these classes have somewhat different behavior, they are
     ! distinguished explicitly here.
```

Comment:
- [ ] Thank you for removing the now-unused LEVGRND_CLASS parameters below. Can you please also remove the now-obsolete comment above (lines 185 - 200)?